ignore_in_scan.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I would prefer a file in the style of `.gitignore` (https://git-scm.com/docs/gitignore)